### PR TITLE
script[setup]: interfaces/input-rich-text-html

### PR DESCRIPTION
**Pull request by rijkvanzanten**

### CODE REVIEW

**Comment by rijkvanzanten** [comment]
CleanShot 2023-05-02 at 18 09 39@2x

I don't trust this. @paescuj mind reviewing?

File: app/src/interfaces/input-rich-text-html/input-rich-text-html.vue
Comment (hanneskuettner) [inline app/src/interfaces/input-rich-text-html/input-rich-text-html.vue:1]: Looks like a lot of the setup function got duplicated and moved into the `internalValue` computed.

Comment (paescuj) [reply app/src/interfaces/input-rich-text-html/input-rich-text-html.vue:1]: Exactly, I remember those 👍 And I tested this component manually ✅

Comment (paescuj) [reply app/src/interfaces/input-rich-text-html/input-rich-text-html.vue:1]: Ah wait a minute, I misread your answer... 😄 I'm going through this pull request in a few minutes 👍

**Comment by paescuj** [comment]
> CleanShot 2023-05-02 at 18 09 39@2x
> 
> I don't trust this. @paescuj mind reviewing?

Result of failed merge commit https://github.com/directus/directus/commit/cfa9b0e0c6f27e9cdeaef13c37cda47445b8c346#diff-59952d1bc4070aa1a416280881f8e270e7cc180f8e87abc026b1e86d438d94c2 🙈

File: app/src/interfaces/input-rich-text-html/input-rich-text-html.vue [hunk 1]
```diff
@@ -211,328 +212,258 @@ type CustomFormat = {
...
+		customFormats?: CustomFormat[];
+		tinymceOverrides?: Record<string, unknown>;
+		disabled?: boolean;
+		imageToken?: string;
+		folder?: string;
+		softLength?: number;
+		direction?: string;
+	}>(),
+	{
+		value: '',
```
Comment (paescuj) [inline app/src/interfaces/input-rich-text-html/input-rich-text-html.vue:230]: I think we can omit this default:
```suggestion
```

**Review by paescuj — APPROVED** [review]

**Pull request merged**